base/range.jl

```diff
@@ -47,23 +47,43 @@ function _colon(start::T, step, stop::T) where T
 end
 
 """
-    range(start[, stop]; length, stop, step=1)
-
-Given a starting value, construct a range either by length or from `start` to `stop`,
-optionally with a given step (defaults to 1, a [`UnitRange`](@ref)).
```

Comment:
`step` does not default to 1 when `length` is specified.

`step` does not default to 1 when `stop` is a positional argument. `range(1,5)` does not work but `range(1,5, step=1)` does. 

The only time `step` defaults to 1 is when using `stop` as a positional argument: `range(1,stop=5)`